### PR TITLE
Move more APIs from libpe_status/libpe_rules to libcrmcommon

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , can you review this when you get a chance? Low priority

File: include/crm/common/action_relation_internal.h [hunk 1]
```diff
@@ -148,4 +148,32 @@ typedef struct {
...
+/*!
+ * \internal
+ * \brief Set action relation flags
+ *
+ * \param[in,out] ar_flags      Flag group to modify
+ * \param[in]     flags_to_set  enum pcmk__action_relation_flags to set
+ */
+#define pcmk__set_relation_flags(ar_flags, flags_to_set) do {           \
+        ar_flags = pcmk__set_flags_as(__func__, __LINE__, LOG_TRACE,    \
+                                      "Action relation", "constraint",  \
```
Comment (nrwahl2) [inline include/crm/common/action_relation_internal.h:160]: I don't know if it's always a constraint but I don't have an immediate better target string idea and this is only for tracing, so meh.

File: lib/common/resources.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 the Pacemaker project contributors
```
Comment (nrwahl2) [inline lib/common/resources.c:2]: 2024

**Review by nrwahl2 — COMMENTED** [review]
Looks good, year is the only thing

File: lib/common/resources.c (hunk 2 above)
Comment (kgaillot) [reply lib/common/resources.c]: Whoops, updated. BTW we recently added `make -C devel copyright` which I obviously forgot to run :)

**Review by nrwahl2 — APPROVED** [review]

**Pull request merged**